test/cases/000_config/010_kmod/kmod.yml

```diff
@@ -13,5 +13,13 @@ onboot:
      - /lib/modules:/lib/modules
     capabilities:
      - all
+services:
```

Comment:
yeah I'll try and repro with latest moby. also as we don't have start ordering it's easier to put poweroff in services for the time being and test code in onboot